experimental/transliteration/src/transliterator/mod.rs

```diff
@@ -1211,32 +1176,30 @@ impl<'a> VarTableElement<'a> {
 }
 
 impl<'a> VarTable<'a> {
-    // TODO(#3958): these must be the same as during datagen. Find some place to define them *once*
-    const BASE: u32 = '\u{F0000}' as u32;
-    const MAX_DYNAMIC: u32 = '\u{FFFF0}' as u32;
-    const RESERVED_PURE_CURSOR: u32 = '\u{FFFFB}' as u32;
-    const RESERVED_ANCHOR_START: u32 = '\u{FFFFC}' as u32;
-    const RESERVED_ANCHOR_END: u32 = '\u{FFFFD}' as u32;
-
-    const ENCODE_RANGE: RangeInclusive<char> = '\u{F0000}'..='\u{FFFFD}';
+    #[doc(hidden)]
```

Comment:
These can be public, as `VarTable` is in `provider`, which is perma-unstable. You could also move them into `provider.rs`